runtime/android/core_internal/src/org/xwalk/core/internal/XWalkSettingsInternal.java

```diff
@@ -784,8 +786,12 @@ public void run() {
     }
 
     /**
-     * See {@link android.webkit.WebSettings#getSaveFormData}.
+     * Gets whether the XWalkView saves form data.
+     * @return whether the WebView saves form data
+     * @see #setSaveFormData
```

Comment:
You can run `/xwalk_core_library_docs/index.html` in the out directory to double check